uint8 group_number
---
bool answer_received
string info
